source/hardware/controllers/reloop_mixage.rst

```diff
@@ -0,0 +1,115 @@
+
+======  =========================================  ==========================================================================================================================================================================  ==============================================================================================================================================================================
+Button  Label                                      Function                                                                                                                                                                    SHIFT function
+======  =========================================  ==========================================================================================================================================================================  ==============================================================================================================================================================================
+1       :hwlabel:`-`                               Allows the temporary deceleration (counter-clockwise) of a track                                                                                                            Adjust track key down 
+2       :hwlabel:`+`                               Allows the temporary acceleration (clockwise) of a track                                                                                                                    Adjust track key up 
+3       :hwlabel:`SHIFT`                           By pressing this button all the device's control elemetns recieve a different function. The assignment can be viewed in the SHIFT function column                           -
+4       :hwlabel:`LOOP` / :hwlabel:`IN`            Set beatloop start at current play position and activate beatloop or turn it off if currently active                                                                        Set loop in point
+5       :hwlabel:`-LENGTH+` / :hwlabel:`BEATMOVE`  Turn to halve/double loop length; Turn while pushing down to adjust beatjump size; Press shortly to clear any loop points                                                   Beatjump in or opposed to playback direction. If a loop is active, beatjump the loop in or opposed to playback direction; Press shortly to enter/exit beatloop adjustment mode
+6       :hwlabel:`GAIN`                            Preamplication of a track/deck                                                                                                                                              Same as non shift function
+7       Pitchfader                                 Adjust the pitch or playback speed of the current deck. The LED lights up in the neutral position                                                                           Same as non shift function
```

Comment:
```suggestion
7       Pitchfader                                 Adjust the pitch or playback speed of the current deck. The LED lights up in the neutral position                                                                           Same as non-shift function
```

Reply:
I'm not a native speaker, this just 'feels' correct..
If you decide to pick this, please change the other occurences, too.